src/views/Dashboard/Lab/Account/index.vue

```diff
@@ -361,6 +364,18 @@ export default {
         this.imageUrl = ""
       }
     },
+
+    editInfo() {
+      if (this.isEditable) {
+        this.isEditable = true
+      } else {
+        return this.isEditable = false
+      }
```

Comment:
The `if` statements are weird what are you trying to achieve?

Reply:
if status account is verified, lab cannot change(edit) addres, city, and region.

Reply:
I still don't get it, please change the logic, it's very confusing. Why don't you just return the `this.isEditable` attribute instead?